pyfar/dsp/window.py

```diff
@@ -0,0 +1,378 @@
+import numpy as np
+from scipy import special as sp
+from pyfar import Signal
+
+
+# different window types; for even window lengths window maximum not 1, especially for short windows!! set maximum to 1 somehow?
+def rect(length):
```

Comment:
I think you make more than necessary here. I would suggest to use functions from `https://docs.scipy.org/doc/scipy/reference/signal.windows.html` wherever possible.